package.json

```diff
@@ -2,11 +2,15 @@
   "name": "react-responsive-picture",
   "version": "1.0.1",
   "description": "A future-proof responsive image component that supports latest Picture specification",
-  "main": "./dist/index.js",
+  "main": "./lib/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
```

Comment:
Do we need this? From [what I've read](https://github.com/rollup/rollup/wiki/pkg.module), I think we could just get away with having `module`.